common/include/pcl/register_point_struct.h

```diff
@@ -336,13 +336,13 @@ namespace pcl
 
 // \note: the mpl::identity weirdness is to support array types without requiring the
 // user to wrap them. The basic problem is:
-// typedef float[81] type; // SYNTAX ERROR!
-// typedef float type[81]; // OK, can now use "type" as a synonym for float[81]
+// using type = float[81]; // SYNTAX ERROR!
```

Comment:
Are these modified comments true?

Reply:
```cpp
using type = float[81];
```
Compiles with `using`. The whole block comment can be removed.

Reply:
Only there won't be any explanation for the "`mpl::identity` weirdness" anymore. But that's fine.